lighthouse-core/report/v2/report-generator.js

```diff
@@ -118,7 +118,19 @@ class ReportGeneratorV2 {
       });
 
       const categoryScore = ReportGeneratorV2.arithmeticMean(audits);
-      return Object.assign({}, category, {audits, score: categoryScore});
+
+      // Check certification (if any).
+      let isCertified;
+      if (category.certification) {
+        // Certification and required audits existence checked by Config.
+        const certification = config.certifications[category.certification];
+        isCertified = certification.audits.reduce((result, certAudit) => {
```

Comment:
i'm okay with it here. (as it is reducing down to a single value, rather than pushing items onto an array).

but now that you mention it, every() seems better. :)